src/redux/movies/movies.utils.js

```diff
@@ -0,0 +1,38 @@
+export const changeLikes = (movies, { movieId, updatedMovie }) => {
+  const moviesCopy =  [...movies];
```

Comment:
no need in `moviesCopy` at all, map/find/reduce/filter methods don't affect initial array.